projects/demo-playwright/tests/addon-commerce/input-card-grouped.spec.ts

```diff
@@ -0,0 +1,165 @@
+import {
+    TuiDocumentationPagePO,
+    tuiGoto,
+    TuiInputCardGroupedPO,
+} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+const {describe, beforeEach, use} = test;
+
+describe(`InputCardGrouped`, () => {
+    use({viewport: {width: 1280, height: 800}});
+
+    let documentationPage: TuiDocumentationPagePO;
+    let inputCardGroupedPage: TuiInputCardGroupedPO;
+    let example: Locator;
+    let cardGroupInput: Locator;
+    let cardExpiryInput: Locator;
+    let cleanerIcon: Locator;
+
+    describe(`API`, () => {
+        beforeEach(({page}) => {
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+            inputCardGroupedPage = new TuiInputCardGroupedPO(example);
+            cardGroupInput = inputCardGroupedPage.cardGroupInput;
+            cleanerIcon = inputCardGroupedPage.cleanerIcon;
+        });
+
+        test(`set value and clear after`, async ({page}) => {
+            await tuiGoto(page, `components/input-card-grouped/API`);
+
+            await cardGroupInput.pressSequentially(`1234 4567 8910 1112`);
+            await expect(cardGroupInput).toHaveScreenshot(
+                `01-input-card-grouped-filled.png`,
+            );
+
+            await cleanerIcon.click();
+            await expect(cardGroupInput).toHaveScreenshot(
+                `02-input-card-grouped-cleared.png`,
+            );
+
+            await example.click({
+                force: true,
+                position: {x: 0, y: 0}, // click top left corner, away from field
+            });
+            await expect(cardGroupInput).toHaveScreenshot(
+                `03-input-card-grouped-unfocused.png`,
+            );
+        });
+
+        test(`disabled input card grouped`, async ({page}) => {
+            await tuiGoto(page, `components/input-card-grouped/API?disabled=true`);
+
+            await expect(cardGroupInput).toHaveCSS(`pointer-events`, `none`);
+            await expect(cardGroupInput).toHaveScreenshot(
+                `04-input-card-grouped-disabled.png`,
+            );
+        });
+    });
+
+    describe(`Examples`, () => {
+        let cardCvcInput: Locator;
+
+        beforeEach(async ({page}) => {
+            await tuiGoto(page, `components/input-card-grouped`);
+
+            documentationPage = new TuiDocumentationPagePO(page);
+        });
+
+        test(`input card grouped with validation`, async () => {
+            example = documentationPage.getExample(`#validation`);
```

Comment:
No need to "share" variable as you create this variable in every test.

```suggestion
            const example = documentationPage.getExample(`#validation`);
```

Reply:
(the same for other variables)